templates/primary-store-service.json

```diff
@@ -0,0 +1,417 @@
+                              "valueFrom": {
+                                 "secretKeyRef" : {
+                                    "name" : "${APPLICATION_NAME}",
+                                    "key" : "application-password"
+                                 }
+                              }
+                           },
+                           {
+                               "name": "NUMBER_OF_OWNERS",
+                               "value": "${NUMBER_OF_OWNERS}"
+                           }
+                        ],
+                        "image": "${IMAGE}",
+                        "livenessProbe": {
+                           "exec": {
+                              "command": [
+                                 "/opt/datagrid/bin/livenessProbe.sh"
+                              ]
+                           },
+                           "initialDelaySeconds": 15,
+                           "timeoutSeconds": 10,
+                           "periodSeconds": 20,
+                           "successThreshold": 1,
+                           "failureThreshold": 5
+                        },
+                        "readinessProbe": {
+                           "exec": {
+                              "command": [
+                                 "/opt/datagrid/bin/readinessProbe.sh"
+                              ]
+                           },
+                           "initialDelaySeconds": 17,
+                           "timeoutSeconds": 10,
+                           "periodSeconds": 20,
+                           "successThreshold": 2,
+                           "failureThreshold": 5
+                        },
+                        "name": "${APPLICATION_NAME}",
+                        "ports": [
+                           {
+                              "containerPort": 8080,
+                              "name": "http",
+                              "protocol": "TCP"
+                           },
+                           {
+                              "containerPort": 8888,
+                              "name": "ping",
+                              "protocol": "TCP"
+                           },
+                           {
+                              "containerPort": 11222,
+                              "name": "hotrod",
+                              "protocol": "TCP"
+                           },
+                           {
+                              "containerPort": 11211,
+                              "name": "memcached",
+                              "protocol": "TCP"
+                           }
+                        ],
+                        "volumeMounts": [
+                           {
+                              "name": "srv-data",
+                              "mountPath": "/opt/datagrid/standalone/data"
+                           },
+                           {
+                              "name": "${APPLICATION_NAME}-configuration",
```

Comment:
I'd say this name is too long again. We will again run into https://issues.jboss.org/browse/ISPN-8611  IMO, the APPLICATION_NAME is unnecessary here as the name is already included in the service name anyway.

Reply:
Sure, I can shorten it up.